1.2.Javascript and Web.md~

```diff
@@ -0,0 +1,6 @@
+###জাভাস্ক্রিপ্ট এবং ওয়েব ### 
```

Comment:
One more thing vaiya. I am not writing this on my own. I am following WROX
Beginning Javascript by Paul Wilton and Jeremy McPeak. They said that
learning Javascript is way easier than learning Java. :)

On Wed, Mar 4, 2015 at 9:15 AM, Md.Sabbir Alam <sabbiralam.cse@gmail.com>
wrote:

> Thank you vaiya for your reply. I am going to fixed the typos and
> suggested changes. :)
>
> On Wed, Mar 4, 2015 at 7:56 AM, Abu Ashraf Masnun <
> notifications@github.com> wrote:
>
>> In 1.2.Javascript and Web.md~
>> <https://github.com/howtocode-com-bd/js.howtocode.com.bd/pull/1#discussion_r25746820>
>> :
>>
>> > @@ -0,0 +1,6 @@
>> > +###জাভাস্ক্রিপ্ট এবং ওয়েব ###
>>
>> I guess this is the backup file generated by your text editor. Were you
>> using Gedit? Using a text editor that has live preview for Markdown would
>> help.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/howtocode-com-bd/js.howtocode.com.bd/pull/1/files#r25746820>
>> .
>>
>
>